internal/llrp/device_service.go

```diff
@@ -35,19 +42,23 @@ const (
 	capReadingName = "ReaderCapabilities"
 
 	maxBody = 100 * 1024
+
```

Comment:
`basePath` above is incorrect now
```go
basePath = "/api/v2/device/name/"
```

Reply:
Goes away once we switch to using the Command Client

Reply:
switched to command client